network-api/networkapi/wagtailpages/pagemodels/buyersguide/products.py

```diff
@@ -421,7 +421,7 @@ def average_bin(self):
             }
 
         average_vote = self.average_creepiness
-        mode_bin = round(average_vote / 20) - 1
+        mode_bin = int(average_vote // 20)
```

Comment:
Thanks for catching this bug! I'm wondering if this can be simplified without `int()`? i.e., 

```suggestion
        mode_bin = average_vote // 20
```

Reply:
Could you please remind me where this magical number `20` is coming from? 😅

Reply:
`20` is the bucket size, each bin/bucket has 20 units (`0 - 20`, `20 - 40`, `40 - 60`, `60 - 80` and `80 - 100`). I'll put that as a constant.

We need `int()` here, otherwise Python will cast `average_vote // 20` as a float.